releases/release-1.27/release-notes/release-notes-draft.md

```diff
@@ -428,6 +430,10 @@
 - `node_stage_path` is now set whenever available for expansion during mount ([#115346](https://github.com/kubernetes/kubernetes/pull/115346), [@gnufied](https://github.com/gnufied))
 - `statefulset` status will now be consistent on API errors ([#113834](https://github.com/kubernetes/kubernetes/pull/113834), [@atiratree](https://github.com/atiratree))
 - `tryUnmount` now respects `mounter.withSafeNotMountedBehavior` ([#114736](https://github.com/kubernetes/kubernetes/pull/114736), [@andyzhangx](https://github.com/andyzhangx))
+- Added call function that validates resize policy for in-place pod resize feature. ([#116857](https://github.com/kubernetes/kubernetes/pull/116857), [@vinaykul](https://github.com/vinaykul)) [SIG Apps]
```

Comment:
Since this is a new feature and this PR was a bug fix to the feature, we don't need to add this release note. I'm not sure why the release-note-none didn't get picked up.
Please remove it.
https://github.com/kubernetes/kubernetes/pull/116857#issuecomment-1479652578

Reply:
Ack. Cleaning up. This was manual change done since there was some issue with the krel tooling.

Reply:
@puerco Ack. we are discussing the same on the `sig-release` slack. Will turn that into a Github Issue soon